README.md

```diff
@@ -77,40 +77,40 @@ sudo make package-toolkit REBUILD_TOOLS=y
 popd
 ```
 
-## Clone CBL-MarinerDemo Repo and Extract the Toolkit
+## Clone CBL-MarinerTutorials Repo and Extract the Toolkit
 
-Now clone the CBL-MarinerDemo repo and extract the toolkit to the CBL-MarinerDemo repository.  
+Now clone the CBL-MarinerTutorials repo and extract the toolkit to the CBL-MarinerTutorials repository.  
 
 ```bash
-git clone https://github.com/microsoft/CBL-MarinerDemo.git
-pushd CBL-MarinerDemo
+git clone https://github.com/microsoft/CBL-MarinerTutorials.git
+pushd CBL-MarinerTutorials
 cp ../CBL-Mariner/out/toolkit-*.tar.gz ./
 tar -xzvf toolkit-*.tar.gz
 ```
 
 The toolkit folder now contains the makefile, support scripts and the go tools compiled from the section.  The toolkit will preserve the previously compiled tool binaries, however the toolkit is also able to rebuild them if desired. (Not recommended: set `REBUILD_TOOLS=y` to use locally rebuilt tool binaries during a build).
 
-The remainder of this tutorial assumes you are using CBL-Mariner 2.0.  However, it is possible to build this Demo using the CBL-Mariner 1.0 Release as well.  
+The remainder of this tutorial assumes you are using CBL-Mariner 2.0.  However, it is possible to build this tutorial using the CBL-Mariner 1.0 Release as well.  

```

Comment:
consider "build the same from this tutorial"  or "build the demo project from this tutorial"